docs: epic dotfiles plug (#9)
catppuccin/waybar: README.md

-	Q: **_"Waybar doesn't work with the colors"_**\
	A: Make sure you included the file in the right place and you are using `@COLOR`

> - Q: **_Hey! Can I borrow that bar config?_**\

Reviewer: ```suggestion
- Q: **_How can I make the bar look like the preview image?_**\
```

> - Q: **_How can I make the bar look like the preview image?_**\

Reviewer: ```suggestion
- Q: **_"How can I make the bar look like the preview image?"_**\
```

>   A: Here's [the waybar config](https://github.com/rubyowo/dotfiles/tree/main/config/waybar) for the preview image.

Reviewer: ```suggestion
  A: Here's [the waybar config](https://github.com/rubyowo/dotfiles/tree/f925cf8e3461420a21b6dc8b8ad1190107b0cc56/config/waybar) for the preview image.
```

>   A: Sure, here's [the waybar config](https://github.com/rubyowo/dotfiles/tree/main/config/waybar) in the preview image.

Reviewer: ```suggestion
  A: Here's [the waybar config](https://github.com/rubyowo/dotfiles/tree/main/config/waybar) for the preview image.
```
